evojax/algo/base.py

```diff
@@ -49,3 +50,15 @@ def best_params(self) -> jnp.ndarray:
     @best_params.setter
     def best_params(self, params: Union[np.ndarray, jnp.ndarray]) -> None:
         raise NotImplementedError()
+
+
+class QualityDiversityMethod(NEAlgorithm, ABC):
```

Comment:
NEAlgorithm is already having ABC, so no need to put ABC here?

Reply:
Thanks!
done